fairing/training/native/decorators.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from builtins import super
+from future import standard_library
+standard_library.install_aliases()
+
+import logging
+
+from fairing.training import base
+from .deployment import NativeDeployment
+from .runtime import BasicNativeRuntime
+
+logger = logging.getLogger(__name__)
+
+class Training(base.TrainingDecoratorInterface):
+    """A simple Kubernetes training.
+    
+    Args:
+        namespace {string} -- (optional) here the training should be deployed
+    """
+
+    def __init__(self, namespace=None):
+        self.namespace = namespace
+        self.runs = 1
+    
+    def _validate(self, user_object):
+        """TODO: Verify that the training conforms to what is expected from 
+            a simple training. (probably not HP tuning)"""
+        pass
+
+    def _train(self, user_object):
+        runtime = BasicNativeRuntime()
+        runtime.execute(user_object)
+
+    def _deploy(self, user_object):
+        deployment = NativeDeployment(self.namespace, self.runs)
+        deployment.execute()
+
+
+# class HPTuning(base.TrainingDecoratorInterface):
```

Comment:
is this meant to be commented out?

Reply:
Yes, I would like to open a discussion of how to imlement hyperparameter tuning properly before reintroducing it. There are a bunch of things that weren't right with the previous approach.
I can also remove the code entirely instead of commenting it if it's an issue.